### ISSUE TITLE
fix(vscode): extension caused Git to get stuck in "Scanning folder for git repositories"

### ISSUE THREAD

**Issue by PhilipNell-258** [VeryGoodOpenSource/dart_frog#1059]
**Description**

VS Code source control is stuck with the following message "Scanning folder for git repositories" when opening a Flutter project with the Dart Frog plugin enabled. After disabling, VS Code was working as normal.

**Steps To Reproduce**

1. Ensure that the Dart Frog Plugin is enabled
2. Open any flutter project
3. Go to source control
4. Message: "Scanning folder for git repositories"

**Expected Behavior**

VS code is supposed to load and init the git inside the project directory

**Screenshots**

N/A

**Additional Context**

VS Code version 1.82.2
Windows 11 Pro

**Comment by alestiago** [comment]
Hi @PhilipNell-258 thanks for opening an issue 💙 ! I have a few questions and I'm trying to reproduce this issue.

With "Dart Frog Plugin" are you referring to the "Dart Frog VS Code extension"? Do you see this issue even after deactivating the "Dart Frog VS Code extension"? What version of the "Dart Frog VS Code extension" are you using? What is the folder structure you've opened in VS Code?

---

Following your reproductive steps in a Mac, I've not been able to reproduce this issue. I've done:

1. Created a Flutter project:
```sh
flutter create flutter_counter_app
```

2. Open the new project in VS Code:
```sh
code flutter_counter_app
```

3. Initialised a git repository.

4. Committed the changes. Didn't see any issue.

5. Open and closed VS Code. Went to source control, didn't see any issue.

**Comment by PhilipNell-258** [comment]
Yes, I am referring to the Dart Frog VS Code Extension. I am currently using v0.2.2. 

The folder structure is a single directory (basically the counter app).

**Comment by alestiago** [comment]
Thanks @PhilipNell-258 , may I ask what Dart Frog CLI version are you running? Do you have the latest version? You can check this running `dart_frog --version` in your terminal.

**Comment by PhilipNell-258** [comment]
@alestiago, I have version 1.1.1 of the Dart Frog CLI installed

**Comment by alestiago** [comment]
@PhilipNell-258 this issue was automatically closed by GitHub since the Pull Request that solves this issue (#1064) has now been merged. You will be able to experience the fix in the next version (0.2.3). 

If in the meantime you would like to try it you can:

1. Clone the repository
2. Get dependencies (from: extension/vscode):
```sh
npm i
```
3. Package the current extension (from: extension/vscode):
```sh
vsce package 0.2.3
```
4. Open VS Code and run from the command palette "Extensions: Install from VSIX" command and select the generated file from step 3.

**Comment by alestiago** [comment]
@PhilipNell-258 the new version ([0.2.3](https://marketplace.visualstudio.com/items?itemName=VeryGoodVentures.dart-frog)) is now released, you should no longer experience this issue. Let me know if you encounter any other issue 🙌 . Thanks for contributing! 💙
